Gopkg.toml

```diff
@@ -32,7 +32,7 @@
 
 [[constraint]]
   name = "github.com/influxdata/influxql"
-  revision = "145e0677ff6418fa00ee7e5dd434305631ab44ea"
+  revision = "8d2756c43bb02f5d3e505c51cf83a1ac9234c419"
```

Comment:
We should change this to `branch = master`.

Reply:
This is always set to an exact revision so the same revision is inherited by the closed source version.

Reply:
I've removed this constraint. I'll create something in the closed source to watch for these kinds of things.